pkg/model/sites.go

```diff
@@ -1,13 +1,46 @@
 package model
 
+import (
+	"fmt"
+)
+
+type ConvertBool bool
+
 // SitesYml is used to map domains across environments for search and replace with WPMS sites.
 type SitesYml struct {
-	APIVersion int        `yaml:"api_version"`
-	DomainMaps DomainMaps `yaml:"domain_maps"`
+	APIVersion            int         `yaml:"api_version"`
+	ConvertToSubdirectory ConvertBool `yaml:"convert_to_subdirectory"`
+	DomainMaps            DomainMaps  `yaml:"domain_maps"`
 }
 
 // DomainMaps is a collection of site ID/site domains keyed by environment name.
 type DomainMaps map[string]DomainMapByEnvironment
 
 // DomainMapByEnvironment is a collection of site domains keyed by site ID.
 type DomainMapByEnvironment map[int]string
+
+// A custom Unmarshal method for ConvertBool, accepting true, false, 1, or 0
+// for convert_to_subdirectory.
+func (convert *ConvertBool) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var val interface{}
+	if err := unmarshal(&val); err != nil {
+		return err
```

Comment:
This err dropped my code coverage, any arbitrary unmarshal I could try in tests was throwing after this functions runs.